packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -1,3 +1,7 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
This should probably be a different year.

Reply:
Done. 2018.